crates/wash-lib/src/cli/mod.rs

```diff
@@ -240,9 +240,12 @@ impl TryFrom<CliConnectionOpts> for WashConnectionOptions {
             ctl_seed,
             ctl_credsfile,
             js_domain,
-            lattice_prefix,
-            timeout_ms,
+            lattice_prefix: match lattice_prefix {
```

Comment:
I don't think we should override just `lattice_prefix` and not the other fields which can be set via context (e.g. `ctl_host`). Since the current struct has `ctx` as a separate field, let's leave that as-is. Instead, I would suggest we define a `get_lattice_prefix` on `WashConnectionOptions`, which could look something like:

```rust
pub fn get_lattice_prefix(&self) -> String {
    self.lattice_prefix.clone().unwrap_or_else(|| {
        self.ctx
            .as_ref()
            .map(|c| c.lattice_prefix.clone())
            .unwrap_or(DEFAULT_LATTICE_PREFIX.to_string())
    })
}
```

Reply:
i made some [changes ](https://github.com/wasmCloud/wasmCloud/pull/954/commits/f4807e8fe4b8be6099259bdb89678b60245e52ff) based on your feedback. let me know if it works.

Reply:
actually nvm... need to refactor to get tests passing